src/plopp/functions/common.py

```diff
@@ -117,4 +126,24 @@ def preprocess(obj: Union[ndarray, Variable, DataArray],
         out = out[dim, start:start + width + 2]
     if not ignore_size:
         _check_size(out)
+    if coords is not None:
+        renamed_dims = {}
+        if isinstance(coords, str):
+            coords = [coords]
```

Comment:
Is this to support `coords='x2'`? If so, why is `coords=x2` not supported as well?

Reply:
It was both to support `coords='x2'` but also to avoid weird errors if people do write `coords='x2'` and the iterator starts iterating over each letter in the dim string, saying `dimension 2 is not found` or something like that.
But maybe I should just raise an error if it is not a list?
Or also support `coords=x2`. Not sure what's best.